src/main/java/us/kilroyrobotics/Robot.java

```diff
@@ -114,7 +120,8 @@ public void autonomousExit() {}
     @Override
     public void teleopInit() {
         Shuffleboard.selectTab("Teleop");
-        m_robotContainer.wristSetCoralStation.schedule();
+        m_robotContainer.wrist.setAngle(CoralMechanismConstants.kIntakingAngle);
+        m_robotContainer.coralIntakeMotor.setSpeed(CoralMechanismConstants.kWheelSpeedHolding);
```

Comment:
Why are we doing this here instead of in the state machine

Reply:
See above.